model/app.py

```diff
@@ -458,7 +467,10 @@ def __init__(self, model, message, cache=None):
                 self.flux = {}
                 self.growth = 0.0
             else:
-                self.flux = solution.data_frame.T.to_dict()
+                self.flux = {
```

Comment:
sympy has some `Float` and `Zero` objects which cannot get serialised, that's the hack for that until optlang fixes it